components/raftstore/src/store/async_io/read.rs

```diff
@@ -141,11 +168,92 @@ where
                     },
                 );
             }
-            ReadTask::GenTabletSnapshot { region_id, .. } => {
-                // TODO: implement generate tablet snapshot for raftstore v2
-                self.notifier
-                    .notify_snapshot_generated(region_id, Box::new(RaftSnapshot::default()));
+
+            ReadTask::GenTabletSnapshot {
```

Comment:
In v1,  the similar logic is in worker/region.rs.  And async_io/read.rs is shared between v1 and v2 too.
I'm wondering if it's better to have this logic in region.rs too.

Reply:
it changed a lot. we discussed that unify snapshot handle to async workers in v2. see https://github.com/tikv/tikv/pull/13555#issuecomment-1296563953.

Reply:
> it changed a lot. we discussed that unify snapshot handler to async workers in v2. see [#13555 (comment)](https://github.com/tikv/tikv/pull/13555#issuecomment-1296563953).

I share the same worry as Bokang that mixing them together could impact raft log fetch or vice versa. For now I'm OK with that. May need to revisit it if performance is impacted by this later.